packages/headless/config/stripe/stripe.ts

```diff
@@ -0,0 +1,5 @@
+export const stripeConfig = {
+  stripe: {
+    publishableKey: 'pk_test_TYooMQauvdEDq54NiTphI7jx',
+  },
```

Comment:
is stripe workable now? or this is previous projects code?

Reply:
removed